app/components/Stepper.tsx

```diff
@@ -0,0 +1,59 @@
+import { Link, useLocation } from '@remix-run/react'
+import React from 'react'
+
+const steps = [
+  { label: 'Add your resume' },
+  { label: 'Add job offer' },
+  { label: 'Enhance resume' },
+]
+
+export default function Stepper() {
+  const location = useLocation()
+  const currentStep = location.pathname.split('/').pop() // e.g. "step1", "step2"
+
+  return (
+    <div className="flex items-center justify-center w-full max-w-md mx-auto mb-8">
+      {steps.map((step, index) => (
+        <React.Fragment key={index}>
```

Comment:
**suggestion**: Consider adding aria-labels to the links for better accessibility.